tobago-core/src/main/java/org/apache/myfaces/tobago/internal/component/AbstractUILinks.java

```diff
@@ -19,12 +19,28 @@
 
 package org.apache.myfaces.tobago.internal.component;
 
+import org.apache.myfaces.tobago.component.SupportsDisabled;
 import org.apache.myfaces.tobago.layout.Orientation;
 
 /**
  * {@link org.apache.myfaces.tobago.internal.taglib.component.LinksTagDeclaration}
  */
-public abstract class AbstractUILinks extends AbstractUIPanelBase {
+public abstract class AbstractUILinks extends AbstractUIPanelBase implements SupportsDisabled {
+
+  enum PropertyKeys {
+    disabled,
+  }
 
   public abstract Orientation getOrientation();
+
+  @Override
+  public boolean isDisabled() {
```

Comment:
I think getter/setter for disabled could be generated. So it's no need to implement it here.